Tests fail when using `AIRFLOW_VAR_INGESTION_LIMIT=<anything-but-1000>`

## Description
<!-- Concisely describe the bug. Compare your experience with what you expected to happen. -->
<!-- For example: "I clicked the 'submit' button and instead of seeing a thank you message, I saw a blank page." -->
PR #555 added an `ingestion_limit` Airflow Variable that could be used to set a (rough) limit on how many records could be ingested for a given provider DAG run. Adding an ingestion limit using environment variables with any value that is not 1000 causes tests to fail.

```
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_build_query_param_default ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[gw0] linux -- Python 3.10.1 /usr/local/bin/python

    def test_build_query_param_default():
        actual_param = clm.get_next_query_params({})
        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
>       assert actual_param == expected_param
E       AssertionError: assert {'cc': '1', '...00, 'skip': 0} == {'cc': '1', '...00, 'skip': 0}
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'limit': 300} != {'limit': 1000}
E         Use -v to get the full diff

tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:50: AssertionError

 tests/dags/providers/provider_api_scripts/test_cleveland_museum.py ⨯                                                                                                                                  8% ▊         

――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_build_query_param_increments_offset ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[gw0] linux -- Python 3.10.1 /usr/local/bin/python

    def test_build_query_param_increments_offset():
        previous_query_params = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
    
        actual_param = clm.get_next_query_params(previous_query_params)
        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 1000}
>       assert actual_param == expected_param
E       AssertionError: assert {'cc': '1', '..., 'skip': 300} == {'cc': '1', '... 'skip': 1000}
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'skip': 300} != {'skip': 1000}
E         Use -v to get the full diff

tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:58: AssertionError

 tests/dags/providers/provider_api_scripts/test_cleveland_museum.py ⨯✓✓✓✓✓✓✓✓✓✓✓                                                                                                                     100% ██████████
============================================================================================= short test summary info ==============================================================================================
FAILED tests/dags/providers/provider_api_scripts/test_cleveland_museum.py::test_build_query_param_default - AssertionError: assert {'cc': '1', '...00, 'skip': 0} == {'cc': '1', '...00, 'skip': 0}
FAILED tests/dags/providers/provider_api_scripts/test_cleveland_museum.py::test_build_query_param_increments_offset - AssertionError: assert {'cc': '1', '..., 'skip': 300} == {'cc': '1', '... 'skip': 1000}

Results (4.92s):
      11 passed
       2 failed
         - tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:47 test_build_query_param_default
         - tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:53 test_build_query_param_increments_offset
error: Recipe `_mount-tests` failed on line 67 with exit code 1
error: Recipe `test` failed on line 75 with exit code 1
```

## Reproduction
1. Add `AIRFLOW_VAR_INGESTION_LIMIT=300` to your `.env` file
1. Run `just test`
1. Observe errors

## Resolution
<!-- Replace the [ ] with [x] to check the box. -->
- [ ] 🙋 I would be interested in resolving this bug.

--- a/justfile
+++ b/justfile
@@ -75,6 +75,7 @@ _deps:
 @_mount-tests command: _deps
     # The test directory is mounted into the container only during testing
     docker-compose {{ DOCKER_FILES }} run \
+        -e AIRFLOW_VAR_INGESTION_LIMIT=1000000 \
         -v {{ justfile_directory() }}/tests:/usr/local/airflow/tests/ \
         -v {{ justfile_directory() }}/pytest.ini:/usr/local/airflow/pytest.ini \
         --rm \

